docker/cuda/Dockerfile

```
@@ -28,14 +28,18 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 # Install PyTorch
 ARG TORCH_CUDA=cu124
-ARG TORCH_VERSION=stable
+ARG TORCH_RELEASE_TYPE=stable
+ARG TORCH_VERSION
```

docker will keep warning us for not setting a default value here
```suggestion
ARG TORCH_VERSION=""
```